aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1731,6 +1743,7 @@ impl AptosVM {
             FeeStatement::zero(),
             ExecutionStatus::Success,
             &get_or_vm_startup_failure(&self.storage_gas_params, log_context)?.change_set_configs,
+            TransactionAuxiliaryData::default(),
```

Comment:
Can we factor out the code for `get_transaction_output` for system transactions? It is always
```rust
let output = get_transaction_output(
  session,
  FeeStatement::zero(),
  ExecutionStatus::Success,
  &get_or_vm_startup_failure(&self.storage_gas_params, log_context)?.change_set_configs,
  TransactionAuxiliaryData::default(),
)?:
```

Reply:
refactored